test/lock_retrier_test.rb

```diff
@@ -50,6 +52,22 @@ def test_with_retries_no_transaction
     assert_equal 1, $migrate_attempts
   end
 
+  def test_lock_timeout_accepts_command_parameter
```

Comment:
I'm not sure this test belongs here but it's ok for the draft PR. I can refactor it to a better location after review.